pkg/azuredisk/azuredisk_v2.go

```diff
@@ -181,6 +187,14 @@ func (d *DriverV2) Run(endpoint, kubeConfigPath string, testBool bool) {
 	})
 
 	ctx, cancel := context.WithCancel(context.Background())
+	c := make(chan os.Signal, 2)
+	signal.Notify(c, []os.Signal{os.Interrupt, syscall.SIGTERM}...)
+	go func() {
+		<-c
+		cancel()
+		<-c
```

Comment:
Won't this require a second SIGINT or SIGTERM to be sent?

Reply:
First signal would cancel the context, and that should be sufficient to allow controller manager to gracefully shut down and exit the process as the controller manager runs on that context. If second signal is fed to the channel, then it will exit the process nonetheless.